usr/bin/nxdk-configure

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+. ${NXDK_DIR}/usr/share/nxdk/prelude.inl
+
```

Comment:
This should have a warning that it's experimenal and might be removed again.

There are other approaches one could take for autoconf (like making `nxdk-autoconf` or manually crafting a `./configure` for each port - this is only 20 trivial lines anyway)